src/_includes/docs/install/reqs/macos/base.md

```diff
@@ -7,8 +7,6 @@
 {% assign v-target = include.target %}
 {% endcase %}
 
-{% render docs/install/admonitions/install-in-order.md %}
```

Comment:
Do we use `docs/install/admonitions/install-in-order.md` anywhere else or can we remove it?

Reply:
We can remove it, sure. I'll add it to this PR.